src/Foundation/UninstalledSite.php

```diff
@@ -108,7 +108,10 @@ private function getIlluminateConfig()
                 'lifetime' => 120,
                 'files' => $this->paths['storage'].'/sessions',
                 'cookie' => 'session'
-            ]
+            ],
+            'view' => [
+                'paths' => [],
+            ],
```

Comment:
Can you clarify the need for these @franzliedke ?

Reply:
Of course! I should have explained that in the commit message, right?

The problem is [this line in Laravel's `ServiceProvider` class](https://github.com/laravel/framework/blob/5.7/src/Illuminate/Support/ServiceProvider.php#L84). When the key does not exist, this now raises a notice in PHP 7.4. Since several of the service providers we use call `loadViewsFrom()`, this affected the installer.